### PR TITLE
Moderinzed images and added text differentiating the Tripal content

### PR DESCRIPTION
**Pull request by spficklin**
# Documentation

Issue #602

## Description

This addresses the request by @laceysanderson to clarify the difference between the **Find Content** and **Find Tripal Content** as well as the **Add Content** and **Add Tripal Content**.  This PR updates the images and adds a blurb of text about the Tripal part.

## Screenshots
content1
content2
content3

### CODE REVIEW

File: docs/user_guide/install_tripal/upgrade_from_tripal2.rst [hunk 1]
```diff
@@ -184,8 +184,25 @@ You have now completed the migration process and can safely disable the Tripal v
...
+
+    Uncaught exception thrown in shutdown function. PDOException: SQLSTATE[54001]: Statement too complex: 7 ERROR:  stack depth limit exceeded
+    HINT:  Increase the configuration parameter &amp;quot;max_stack_depth
+
+
+You can avoid this problem by clearing out the Drupal search tables byu executing the following SQL commands:
+
+.. code-block:: sql
+
+    db_query('TRUNCATE {search_total}')
```
Comment (almasaeed2010) [inline docs/user_guide/install_tripal/upgrade_from_tripal2.rst]: No need for db_query in sql.
```suggestion
    TRUNCATE search_total;
```

File: docs/user_guide/install_tripal/upgrade_from_tripal2.rst [hunk 2]
```diff
@@ -184,8 +184,25 @@ You have now completed the migration process and can safely disable the Tripal v
...
+    db_query('TRUNCATE {search_index}')
```
Comment (almasaeed2010) [inline docs/user_guide/install_tripal/upgrade_from_tripal2.rst]: Same issue as above.

```suggestion
    TRUNCATE search_index;
```

**Review by almasaeed2010 — CHANGES_REQUESTED** [review]
Everything looks good to me except for the simple fixes in sql.

Thanks @spficklin

**Comment by spficklin** [comment]
Oh good catch @almasaeed2010 .  Thanks for the review.

**Pull request merged**